frontend/src/components/UserSummaryCard/index.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
+import { isAxiosError } from 'axios';
+import { useQuery } from '@tanstack/vue-query';
+import Card from 'primevue/card';
+import { AppActlApiService } from '@/services/ApiServiceFactory';
+import { DEFAULT_COL_SIZE } from '@/components/UserSummaryCard/constants';
+import CardTextCol from '@/components/CardTextCol/index.vue';
+
+const props = defineProps<{
+    userId: string; // Fam User ID
+    applicationId: string
+}>();
+
+const userQuery = useQuery(
+    {
+        queryKey: ['fam_applications', props.applicationId, 'users', props.userId],
```

Comment:
There’s no risk of collision here. If two queries have the same queryKey, they are considered the same API call by TanStack Query, which is intentional behavior. The queryKey is what TanStack Query uses to identify and deduplicate queries, so having the same key would only happen if they are meant to represent the same data. In this case, the queryKey uniquely represents user data within an application, which avoids unintended overlaps.

Reply:
My initial confusion was for the naming of the key for the purpose of this query.
For developers new to it might not know what the convention for the key should be, would be great somewhere to link to the convention so others don't create the wrong keys. It is ok if you think is should not have issue.